src/vstart.sh

```diff
@@ -991,8 +1003,21 @@ EOF
         ceph_adm config-key set mgr/cephadm/ssh_identity_key -i ~/.ssh/id_rsa
         ceph_adm config-key set mgr/cephadm/ssh_identity_pub -i ~/.ssh/id_rsa.pub
         ceph_adm mgr module enable cephadm
+        ceph_adm config set mgr mgr/cephadm/log_to_cluster_level debug
+        if [ -n "$cephadm_ssh_config_path" ]; then
+            ceph_adm cephadm set-ssh-config -i "$cephadm_ssh_config_path"
```

Comment:
I'd put that before enabling `cephadm` and directly insert the config into the store.